### DISCUSSION TITLE
Always on Display not working while connected to Android Auto

### DISCUSSION THREAD

**Discussion by Pentagit** [NightscoutFoundation/xDrip#3402] [Bug reports]
xDrips Always on Display immediately stops appearing as soon as the smartphone gets connected to Android Auto via USB..

AOD works fine in every other situation. Esp. while charging normally on usb-c-power supply at home.
xDrip has full control over device.

xDrip version: 2124031017 , e5113d9-2024.03.10
Android Auto version: 11.4.640814-release
phone: Pixel 8 Pro
Car: Volvo
Connection: USB-C-Cable

**Comment by Navid200** [comment]
When you use Android auto, does anything else appear on the phone screen?
I mean isn't Android auto meant to be used when you want to use the car screen?

**Reply by Pentagit** [reply]
Most information is being split, e.g. appears on my phone screen and on AndroidAuto at the same time (maps, spotify, android aps notifications, etc). But that xdrip AOD just disappears as long as the smartphone is connected to my car. I have my phone right beside my car display on a phone holder and it would be great if my bg would be displayed on the phone screen while driving.

**Comment by Navid200** [comment]
@jamorham How the phone behaves with respect to always on display is under the control of Android and not xDrip, right?
Or, is there anything we can do in xDrip so that AOD remains active even when the phone is connected to the car for Android Auto?

**Comment by Navid200** [comment]
@Pentagit If you uninstall xDrip and enable Always On Display so that the phone shows time when the screen is locked and then use Android Auto, does your phone still show the time on its screen?

**Reply by Pentagit** [reply]
@Navid200 Yes. I just tested by disabling AOD and removing the permission for xdrip. The phone AOD including time stays exactly the same when connecting to Android Auto. Time shows on phone screen with and without connection to android Auto.

**Reply by Navid200** [reply]
What do you mean by disabling AOD?  How would that be a verification that this is related to xDrip if you disabled AOD?

**Reply by Pentagit** [reply]
I was talking about the xdrip aod there, not the phones aod

**Reply by Navid200** [reply]
Thanks for trying.
Do you have any other app on the phone that can show on AOD?
If yes, can you test with that?  Does that show when Android Auto is active?

**Reply by Pentagit** [reply]
I randomly installed some aod app 
(https://play.google.com/store/apps/details?id=com.sparkine.muvizedge) for testing.

The aod (from that app) stays ON when connecting to Android auto. BUT when unlocking phone , it will not appear again as long as the phone is connected. Weird.

**Comment by Navid200** [comment]
I cannot make any promises, and I may close the issue if it turns out that we cannot do anything in xDrip to address this. 
Please click on create issue from discussion to convert this to an issue.

**Reply by Pentagit** [reply]
Done, thank you!
